packages/client/tests/functional/relationMode/tests_1-to-1.ts

```diff
@@ -0,0 +1,1413 @@
+      const userModel = 'userOneToOne'
+      const profileModel = 'profileOneToOne'
+      const profileColumn = 'profile'
+
+      beforeEach(async () => {
+        await prisma.$transaction([prisma[profileModel].deleteMany(), prisma[userModel].deleteMany()])
+      })
+
+      describe('[create]', () => {
+        testIf(isRelationMode_prisma)(
+          'relationMode=prisma [create] child with non existing parent should succeed',
+          async () => {
+            await prisma[profileModel].create({
+              data: {
+                id: '1',
+                userId: '1',
+              },
+            })
+
+            expect(
+              await prisma[profileModel].findMany({
+                orderBy: [{ id: 'asc' }],
+              }),
+            ).toEqual([
+              {
+                id: '1',
+                userId: '1',
+                enabled: null,
+              },
+            ])
+          },
+        )
+        testIf(isRelationMode_foreignKeys)(
+          'relationMode=foreignKeys [create] child with non existing parent should throw',
+          async () => {
+            await expect(
+              prisma[profileModel].create({
+                data: {
+                  id: '1',
+                  userId: '1',
+                },
+              }),
+            ).rejects.toThrowError(
+              conditionalError.snapshot({
+                foreignKeys: {
+                  [Providers.POSTGRESQL]:
+                    'Foreign key constraint failed on the field: `ProfileOneToOne_userId_fkey (index)`',
+                  [Providers.COCKROACHDB]: 'Foreign key constraint failed on the field: `(not available)`',
+                  [Providers.MYSQL]: 'Foreign key constraint failed on the field: `userId`',
+                  [Providers.SQLSERVER]:
+                    'Foreign key constraint failed on the field: `ProfileOneToOne_userId_fkey (index)`',
+                  [Providers.SQLITE]: 'Foreign key constraint failed on the field: `foreign key`',
+                },
+              }),
+            )
+
+            expect(
+              await prisma[profileModel].findMany({
+                orderBy: [{ id: 'asc' }],
+              }),
+            ).toEqual([])
+          },
+        )
+
+        test('[create] child with undefined parent should throw with type error', async () => {
+          await expect(
+            prisma[profileModel].create({
+              data: {
+                id: '1',
+                // @ts-expect-error
+                userId: undefined, // this would actually be a type-error, but we don't have access to types here
+              },
+            }),
+          ).rejects.toThrowError('Argument user for data.user is missing.')
+
+          expect(
+            await prisma[profileModel].findMany({
+              orderBy: [{ id: 'asc' }],
+            }),
+          ).toEqual([])
+        })
+
+        test('[create] nested child [create]', async () => {
```

Comment:
Is this title missing the expectation of the test? (Seems it should universally succeed)